src/qhttp/AjaxEndpoint.h

```diff
@@ -58,7 +58,9 @@ class AjaxEndpoint
 
 private:
 
-    AjaxEndpoint();
+    AjaxEndpoint(std::shared_ptr<Server> server);
+
+    std::shared_ptr<Server> _server;
```

Comment:
Due to a design (static construction, no copy semantic) of the class this needs to be:
```
std::shared_ptr<Server> const _server;
```

Reply:
Will do